versions.json

```diff
@@ -79,6 +79,6 @@
     },
     "support-bundle-kit": {
         "repo": "https://github.com/rancher/support-bundle-kit.git",
-        "tag": "v0.0.75"
+        "tag": "v0.0.76"
```

Comment:
Dependent on the release of https://github.com/rancher/support-bundle-kit/pull/162.